reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-70.md

```diff
@@ -0,0 +1,809 @@
+---
+title: What's New in PowerShell 7.0
+description: New features and changes released in PowerShell 7.0
+ms.date: 02/19/2020
+---
+
+# What's New in PowerShell 7.0
+
+We are happy to announce the Generally Available (GA) release of PowerShell 7.0! First! We'd like to
```

Comment:
First, we'd like to...

Reply:
Hey @alexandair - Thanks for the help!

Reply:
You're welcome.

Reply:
@theJasonHelmick 

There is one type of a problem that I didn't comment on so far, because there are a lot of occurrences of it.  

If you look at the other PowerShell documentation articles the rule is to write:

the Select-Object cmdlet ("the cmdlet Select-Object" is incorrect)
the WINDIR environment variable ("the environment variable WINDIR" is incorrect)